src/site/xdoc/index.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<document>
+ <properties>
+  <title>Overview</title>
+  <author email="dev@commons.apache.org">Commons Documentation Team</author>
+ </properties>
+<body>
+<!-- ================================================== -->
+<section name="Apache Commons Crypto">
+    <p>Apache Commons Crypto is a cryptographic library optimized with AES-NI (Advanced Encryption
+        Standard New Instructions). It provides Java API for both cipher level and Java stream
+        level. Developers can use it to implement high performance AES encryption/decryption with
+        the minimum code and effort. Please note that Apache Commons Crypto doesn't implement the
+        cryptographic algorithm such as AES directly. It wraps to Openssl or JCE which implement the
+        algorithms.
+    </p>
+</section>
+<!-- ================================================== -->
+<section name="Features">
+  <p>
+    <ul>
+      <li>Cipher API for low level cryptographic operations.</li>
+      <li>Java stream API (CryptoInputStream/CryptoOutputStream) for high level stream
+        encyrption/decryption.
+      </li>
+      <li>Both optimized with high performance AES encryption/decryption. (1400 MB/s - 1700 MB/s
```

Comment:
`Both optimized with high performance AES encryption/decryption. (1400 MB/s - 1700 MB throughput in modern Xeon processors).
`
Change to: 
`High performance AES encryption/decryption optimized with Intel AES-NI.`